backend/src/gameSessions.ts

```diff
@@ -0,0 +1,62 @@
+import { Game } from "./entities/game";
+import { logger } from "./utils/logger";
+import { getRepository } from "typeorm";
+import { getNRandom } from "./utils";
+
+export class GameSessions {
+  private readonly games: { [key: string]: Game };
+
+  constructor() {
+    this.games = {};
+  }
+
+  public new(game: Game): void {
+    this.games[game.title] = game;
```

Comment:
Maybe it is better to always have an id on the game so that we can have 2 games with the same title.

Reply:
Done